frontend/src/scalars/ui/Config.vue

```diff
@@ -29,10 +29,9 @@
                 v-model="config.sortingMethod"
                 label="Tooltip sorting method"
                 class="visual-dl-page-config-selector"
-                dark
+                dark dense
         ></v-select>
 
-        <v-checkbox class="visual-dl-page-config-checkbox" label="Show data download links" v-model="config.downloadLink" dark></v-checkbox>
```

Comment:
Why do we remove the checkbox? I think we should keep this checkbox so we don't always display the download buttons. Please let me know if the design changes

Reply:
Ya, the data buttons would now be collapsed within each chart